### PR TITLE
Gcc7update

### PR DESCRIPTION
**Pull request by swflb**
@bnhhughes @brobeson I have built this with updates and think all is well. Please review.

### CODE REVIEW

File: Findexternpro.cmake [hunk 1]
```diff
@@ -7,24 +7,18 @@ function(getCompilerPrefix _ret)
   set(options GCC_TWO_VER)
   cmake_parse_arguments(X "${options}" "" "" ${ARGN})
   if(MSVC)
-    if(MSVC14)
+    if(MSVC_VERSION GREATER 1910 AND MSVC_VERSION LESS 1919) # VS 15.0 2017
```
Comment (swflb) [inline Findexternpro.cmake:10]: this file should be a copy/paste from externpro, so changes to this file are a result of updating it to match changes in externpro.

File: projects/README.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
+|[cppzmq](https://github.com/zeromq/cppzmq 'cppzmq')|[MIT](https://github.com/zeromq/cppzmq/blob/v4.2.2/LICENSE 'MIT')|cppzmq is a minimal c++ binding to the libzmq functions.|4.2.2|[repo](https://github.com/zeromq/cppzmq/tree/v4.2.2 'cppzmq repo on github')|none|
 |[GLEW](http://glew.sourceforge.net/ 'The OpenGL Extension Wrangler Library')|[open](http://glew.sourceforge.net/credits.html 'Modified BSD, Mesa 3-D (MIT), and Khronos (MIT)')|The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.|1.13.0|[repo](https://github.com/nigels-com/glew 'GLEW repo on github')|none|
-|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[patch](../patches/hdf5.patch 'patch')|
+|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.22|none|[patch](../patches/hdf5.patch 'patch')|
```
Comment (bnhhughes) [inline projects/README.md]: This didn't actually get updated did it?  Should revert the revision number back to 1.8.16.

**Review by bnhhughes — APPROVED** [review]
Just need to revert the HDF5 version number back to 1.8.16 since you mentioned 1.8.22 had build issues.

**Comment by swflb** [comment]
@bnhhughes thanks for catching inadvertent change in the readme. i just push up a change reverting that.

**Pull request merged**